go.mod

```diff
@@ -110,11 +110,11 @@ require (
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.19.0 // indirect
 	golang.org/x/crypto v0.1.0 // indirect
-	golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10 // indirect
+	golang.org/x/net v0.6.0 // indirect
```

Comment:
I did you explicitly choose 0.6.0 here?

```
❯ go get golang.org/x/net@v0.6.0
go: downloading golang.org/x/net v0.6.0
go: downloading golang.org/x/sys v0.5.0
go: downloading golang.org/x/text v0.7.0
go: downloading golang.org/x/term v0.5.0
go: upgraded golang.org/x/net v0.3.1-0.20221206200815-1e63c2f08a10 => v0.6.0
go: upgraded golang.org/x/sys v0.3.0 => v0.5.0
go: upgraded golang.org/x/term v0.3.0 => v0.5.0
go: upgraded golang.org/x/text v0.5.0 => v0.7.0
```

Reply:
@flavio-fernandes , I just added `golang.org/x/net => golang.org/x/net v0.9.0` in replace directive, then ran `go mod tidy` and `go mod vendor`. Isn't that sufficient ?

Reply:
You mean `v0.6.0`, right? Ack. That worked for me.

Reply:
no, It's `v0.9.0`[here](https://github.com/openshift/sdn/pull/538/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R164), The versions > `v0.7.0` has fix for this particular [CVE](https://pkg.go.dev/vuln/GO-2023-1571), so added latest version `v0.9.0` into replace directive to make upgrade work. The `v0.6.0` is generated from go mod tidy command and i hope this is okay as this might get replaced with `v0.9.0` at compile time.

Reply:
ah, ack. TY for walking me through this.